src/traffic_top/stats.h

```diff
@@ -433,14 +433,22 @@ class Stats
         value *= 100;
       }
     } else if (type == 6 || type == 7) {
-      double numerator;
-      double denominator;
-      getStat(item.numerator, numerator, 2);
-      getStat(item.denominator, denominator, 2);
-      value = numerator + denominator;
+      // add rate
+      double first;
+      double second;
+      getStat(item.numerator, first, 2);
+      getStat(item.denominator, second, 2);
+      value = first + second;
       if (type == 7) {
         value *= 8;
       }
+    } else if (type == 9) {
+      // add
+      double first;
+      double second;
+      getStat(item.numerator, first);
+      getStat(item.denominator, second);
+      value = first + second;
```

Comment:
The clang-analyzer doubt `first` & `second` have garbage value. It's false positive because above `getStat()` calls set values, I think. 

https://ci.trafficserver.apache.org/clang-analyzer/github/6620/2020-04-12-163858-1021-1/report-11e70d.html#EndPath
https://ci.trafficserver.apache.org/clang-analyzer/github/6620/2020-04-12-163858-1021-1/report-063df9.html#EndPath

Reply:
I added a default value for getStat(), to make clang-analyzer happy